Library/ENV/4.3/cc

```diff
@@ -1,6 +1,7 @@
-#!/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby -W0
+#!/usr/bin/ruby -W0
```

Comment:
I'm not sure what was the original intent of using ruby from framework. It would be great to hear what upstream thinks about this.

If framework's ruby is a necessity, we could have a ruby script in the superenv to call a correct ruby version depending on the system.